python/dgl/distributed/partition.py

```diff
@@ -283,7 +283,7 @@ def _verify_graphbolt_partition(graph, part_id, gpb, ntypes, etypes):
     print(f"Partition {part_id} looks good!")
 
 
-def load_partition(part_config, part_id, load_feats=True, use_graphbolt=False):
+def load_partition(part_config, part_id, load_feats=True, use_graphbolt=None):
```

Comment:
`None` as the default value is a bit weird. could we keep the original default value?